src/App.tsx

```diff
@@ -691,9 +705,40 @@ function App(): ReactElement {
       {/* TODO: Split into its own component? */}
       <div className={styles.header}>
         <div className={styles.headerLeft}>
-          <h1>Timelapse Colorizer</h1>
+          <AicsLogoLink href="https://www.allencell.org/" rel="noopener noreferrer">
+            <StyledAicsLogo />
+          </AicsLogoLink>
           <span className={styles.verticalDivider}></span>
+          <h1 style={{ whiteSpace: "nowrap" }}>Timelapse Colorizer</h1>
+        </div>
+        {/* <h3>Dataset Name</h3> */}
```

Comment:
Is this commented element meant to be included?

Reply:
Yes, placeholder for the future dataset names! Will be replaced once the data flow for that is done :')